vllm/model_executor/layers/rotary_embedding.py

```diff
@@ -973,9 +995,7 @@ def get_rope(
             rotary_emb = DeepseekScalingRotaryEmbedding(
                 head_size, rotary_dim, original_max_position, base,
                 is_neox_style, scaling_factor, dtype, **extra_kwargs)
-        # The correct one should be "longrope" but keep "su" here
-        # for backward compatible
-        elif scaling_type == "su" or scaling_type == "longrope":
```

Comment:
Since we update the config in-place inside `vllm/config.py`, there should be no need to normalize this again.